src/Rules/RuleMetaDataProvider.cs

```diff
@@ -55,6 +56,8 @@ public async Task<IRuleInfo> GetRuleInfoAsync(SonarCompositeRuleId ruleId, Cance
 
             //TODO: this does not seem to support taint.
             var language = Language.GetLanguageFromRepositoryKey(ruleId.RepoKey);
+            Debug.Assert(language != null, $"Unable to determine the language from the repo key. RepoKey: {ruleId.RepoKey}");
```

Comment:
Added an assertion to make this kind of failure more obvious during debugging/local testing.

Reply:
FYI previously, `language` would be null which would cause an unhandled exception in line 63 below.

This isn't the only change required to fix the bug - secrets rule info returns null for one of the fields, which causes an different error later. However, I'll create a separate PR for that.